### PR TITLE
Force footer to remain at the bottom

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Before:

Screenshot from 2020-04-06 21-00-52

After:

Screenshot from 2020-04-06 20-58-13

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -166,8 +166,7 @@ pre {
  * Wrapper
  */
 .wrapper {
-    max-width: -webkit-calc(800px - (#{$spacing-unit} * 2));
-    max-width:         calc(800px - (#{$spacing-unit} * 2));
+    max-width: calc(800px - (#{$spacing-unit} * 2));
```
Comment (EPashkin) [inline _sass/_base.scss:169]: @GuillaumeGomez are you tested on chrome-based browser ?
https://stackoverflow.com/questions/39970936/how-to-add-webkit-support-to-calc

Comment (GuillaumeGomez) [reply _sass/_base.scss:169]: On chrome/chromium.

**Comment by EPashkin** [comment]
👍

**Comment by sdroege** [comment]
:+1:

**Pull request merged**